### DISCUSSION TITLE
The border of rectangular selection highlight is not showing up

### DISCUSSION THREAD

**Discussion by sept924** [RyotaUshio/obsidian-pdf-plus#219] [Q&A]
I have updated my PDF++ version to 0.40.5 tonight and after a restart, the border of the rectangular selection highlight in all my PDF is not showing up. The highlight is still there, when I click the highlighted part, it still direct me to the correct notes, but the border is gone.
Not sure if it only happened to me or it happens to others.

**Comment by RyotaUshio** [comment]
Hi @sept924, I've received a similar bug report #220, and it was caused by the installer version (the version of the Obsidian app when you last installed the app using the installer downloaded from https://obsidian.md) being too old (v1.4.13).
Can you try re-installing the Obsidian app using the latest installer (https://obsidian.md/download)? In the case of #220, it solved the issue, so it might also work for your case.
